plugin.video.wnbc/addon.xml

```diff
@@ -20,6 +20,6 @@
     <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
     <forum>http://forum.kodi.tv/showthread.php?tid=234448</forum>
     <website>www.nbc.com</website>
-    <source>https://github.com/learningit/Kodi-plugins-source/tree/master/plugin.video.wnbc</source>
+    <source>https://github.com/learningit/repo-plugins/</source>
```

Comment:
Is this the repo you want contributors to submit their code changes?

Reply:
Yes. I only maintain a single fork repo because it's a terror maintaining separate repos for 60 add-ons in GitHub.